cpp/src/arrow/python/python-test.cc

```diff
@@ -86,7 +86,7 @@ TEST(PandasConversionTest, TestObjectBlockWriteFails) {
 
   PyObject* out;
   Py_BEGIN_ALLOW_THREADS;
-  PandasOptions options;
+  PandasOptions options = {false, false};
```

Comment:
Can you instead add a default ctor to `PandasOptions`?

Reply:
Done!